src/main/kotlin/eu/kevin/api/models/account/AccountRequest.kt

```diff
@@ -0,0 +1,10 @@
+package eu.kevin.api.models.account
+
+internal interface AccountRequest {
+    val accessToken: String
+    val psuIPAddress: String
+    val psuUserAgent: String
+    val psuIPPort: String
+    val psuHttpMethod: PsuHttpMethod
+    val psuDeviceId: String
```

Comment:
Maybe AccountsRequest should have a variable object called AccountsHeaders where of these would be put here? We would then separate headers from custom properties which might be added to the request.